src/components/flatParameters/FlatParameters.tsx

```diff
@@ -12,7 +12,7 @@ import {
     Divider,
     IconButton,
     List,
-    ListItem,
+    ListItemButton,
```

Comment:
Same question

Reply:
Because disabled is deprecated in ListItemBaseProps 
![Capture d’écran du 2024-10-21 11-30-09](https://github.com/user-attachments/assets/288f7b65-20a3-48b6-894d-f90e5a0db373)